app/views/users/view.scala.html

```diff
@@ -141,7 +142,13 @@ <h1 class="username">
 
                     @defining(user.globalRoles.toList.sortBy(_.trust).lastOption) { roleOpt =>
                         @roleOpt.map { role =>
-                            <li class="user-role channel" style="background-color: @role.color.hex">@role.title</li>
+                            @if(users.current.isDefined && users.current.get.can(ViewActivity).in(user)) {
```

Comment:
`users.current.get can ViewActivity in user ` would be more consistent with the rest of the code